Types.ts

```diff
@@ -471,20 +471,28 @@ export enum LinkEventName {
 }
 
 export enum LinkEventViewName {
+    ACCEPT_TOS = 'ACCEPT_TOS',
     CONNECTED = 'CONNECTED',
     CONSENT = 'CONSENT',
     CREDENTIAL = 'CREDENTIAL',
+    DOCUMENTARY_VERIFICATION = 'DOCUMENTARY_VERIFICATION',
     ERROR = 'ERROR',
     EXIT = 'EXIT',
+    KYC_CHECK = 'KYC_CHECK',
+    LIVENESS = 'LIVENESS',
```

Comment:
```suggestion
    SELFIE_CHECK = 'SELFIE_CHECK',
```

This PR adds the view names that are emitted by the IDV product. Without this PR, all events emitted by IDV will not be typed.